tickets/src/components/interface/DatePicker.js

```diff
@@ -27,10 +27,10 @@ export const getDaysMonthsAndYearsForSelect = (now, year, month) => {
 export default class DatePicker extends Component {
   constructor(props) {
     super(props)
-    const { now } = props
+    const { now, date } = props
     this.state = {
       ...getDaysMonthsAndYearsForSelect(now),
-      date: now, // defaults to now
+      date: date < now ? date : now,
```

Comment:
Why don't we leave the past date? This may be confusing for people who edit an old event?